spec/pushmi_pullyu/swift_depositer_spec.rb

```diff
@@ -40,7 +40,7 @@
 
           expect(first_deposit.name).to eq(second_deposit.name)
           expect(first_deposit.container.name).to eq(second_deposit.container.name)
-        end.to change { swift_depositer.swift_connection.container('ERA').count.to_i }.by(1)
+        end.to change { swift_depositer.instance_variable_get('@swift_connection').container('ERA').count.to_i }.by(1)
```

Comment:
I think it was easier to read before, but I agree that having an attribute accessor is unnecessary just for this purpose.